Feature: Add a feature branch

Jolak5 opened the pull request:
In this feature branch, I: 

1. Add a .gitignore file. 
2. Add node_modules to gitignore file. 
3. Set up a linter for HTML and CSS. Add descriptive README file. 
4. Add html file that includes a header with the text "Hello Microverse!". 
5. Add a .css file that includes one class with styles to add blue to header. 
6. Add a test.md file with the text "This file should be ignored by git".

Jolak5 committed:
Include an index.html file

Jolak5 committed:
add the css file that includes heading color

Jolak5 committed:
create a text.md file

Jolak5 committed:
modify the .gitignore file to exclude test.md

Jolak5 committed:
Update ReadME

Added details to the ReadME file about the project

Jolak5 committed:
Create LICENSE.md

I added a license to my repository so that others can use it.

Jolak5 committed:
Update README.md

Jolak5 committed:
Updated the ReadME file

BregornOriginal commented on .gitignore:
```diff
@@ -0,0 +1,6 @@
+# .gitignore
+node_modules/
+
+linters.yml
```
- [ ] Linters are not working on the Pull Request, kindly follow the instructions on the [documentation](https://github.com/microverseinc/linters-config). The linters.yml shouldn't be ignored for instance.

BregornOriginal commented on .gitignore:
```diff
@@ -0,0 +1,6 @@
+node_modules/
+
+linters.yml
+
```
- [ ] Kindly, check the logo of Microverse on the README file, since it's not appearing on it.

![image](https://user-images.githubusercontent.com/93630700/204561318-6db24788-79c6-4c01-bba8-d566ac2a8d41.png)

Jolak5 replied:
Hi, thanks a lot for the review. 

The logo isn't showing because I didn't add the file to my folder. Can I remove the Logo or is it a must I add it?

BregornOriginal commented on .gitignore:
```diff
@@ -0,0 +1,6 @@
+# .gitignore
+node_modules/
+
```
- [ ] Remember that when we create a Pull Request, the person that it's going to review your project needs to know on the first view the changes that you're doing on that PR. Kindly add a description of the changes that you are making on this PR, you can use the requirements of this activity for help 💯

BregornOriginal commented on .gitignore:
```diff
@@ -0,0 +1,6 @@
+
+linters.yml
+
+test.md
```
✅ Good work ignoring node_modules and Test.md files 🥇

BregornOriginal reviewed (changes requested):
## Hi @Jolak5 💯 💯 💯 

## Good job so far!
## There are some issues that you still need to work on to go to the next project but you are almost there!

![one-step-closer-one-more](https://user-images.githubusercontent.com/93630700/204560234-1758b01e-a630-4884-9eb1-e84d886eeb95.gif)

## Highlights

✅ Good use of GitHub Flow
✅ Good commit history!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Jolak5 committed:
Remove the linter file that was added to gitignore

Jolak5 committed:
ensure that the local repo is up to date

Jolak5 committed:
add difference in local repo

Jolak5 committed:
format the styles.css file

Jolak5 committed:
Change the format of the styles.css

Jolak5 committed:
Add the microverse logo

amejid commented on .gitignore:
```diff
@@ -0,0 +1,8 @@
+# .gitignore
+node_modules/
+
```
- [ ] Kindly edit the Pull Request description as requested by the previous reviewer. You mentioned the changes you made to the title instead of the description. 
All the things you mentioned on the title should be here.
![Screenshot from 2022-11-29 19-10-47](https://user-images.githubusercontent.com/17243325/204584751-b3debb20-3156-4e35-930f-46c1416b183c.png)
Please refer [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) to get more information about writing descriptive Pull Requests. ✍️

amejid commented on .gitignore:
```diff
@@ -0,0 +1,8 @@
+node_modules/
+
+<<<<<<< HEAD
+test.md
```
- [OPTIONAL] In the [project requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) it is mentioned that the `test.md` should not be in the Github repository.
You can achieve this by removing the file manually and committing your work once again.

amejid reviewed (changes requested):
Hi @Jolak5, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Linter checks passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Jolak5 committed:
Update .gitignore

Jolak5 committed:
Delete test.md

Deleted text.md in response to the review by the code reviewer and in alignment with the exercise requirement

joskalenda reviewed (approved):
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @Jolak5 , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
| <img src="https://media4.giphy.com/media/TNO6mwK8s38vpHjh8Y/giphy.gif"/>|

### Highlights
✅ Good commit messages

---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
